FlashCap.Core/Devices/DirectShowProperty.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FlashCap.Internal;
+
+namespace FlashCap.Devices
+{
+    public sealed class DirectShowProperty : CaptureDeviceProperty
+    {
+        public DirectShowProperty(VideoProcessingAmplifierProperty property, int min, int max, int step) : base(property, min, max, step) { }
+
+        internal static NativeMethods_DirectShow.VideoProcAmpProperty FromVideoProcessingAmplifierProperty(VideoProcessingAmplifierProperty property)
```

Comment:
each device property should be able to map from `VideoProcessingAmplifierProperty` to their own implementation

Reply:
Perhaps there is a more desirable interface, but this is fine for now 👍